setup.py

```diff
@@ -35,7 +40,10 @@
 
 
 def get_download_url() -> Tuple[str, str]:
-    postfix, sha256 = POSTFIX_SHA256[sys.platform]
+    if platform.machine() == 'aarch64':
+        postfix, sha256 = POSTFIX_SHA256['linuxaarch64']
+    else:
```

Comment:
this defeats the purpose of the mapping lookup -- the mapping should be moved to include architecture if that's the intent

Reply:
Fixed the mapping.